src/Common/src/Interop/Unix/Interop.Errors.cs

```diff
@@ -61,6 +61,7 @@ internal enum Error
         ENETUNREACH      = 0x10028,           // Network unreachable.
         ENFILE           = 0x10029,           // Too many files open in system.
         ENOBUFS          = 0x1002A,           // No buffer space available.
+        ENODATA          = 0x1002B,           // No message is available on the STREAM head read queue.
```

Comment:
Good catch! Most likely caused by incorrect merging in our staging branch.

Still, I believe finding this is lucky: the fix in #2864 should add build-time validations to ensure these are not added again. 
I will update our branch as well but there might be other similar issues.